code/__defines/rust_g_overrides.dm

```diff
@@ -0,0 +1,3 @@
+// RUSTG_OVERRIDE_BUILTINS is not used since the file APIs don't work well over Linux.
+#define url_encode(text) rustg_url_encode("[text]")
+#define url_decode(text) rustg_url_decode("[text]")
```

Comment:
@DameonOwen I believe we're on a Linux box? Up to you if you think this will cause any issues, but I assume the other server has this enabled so it's probably nothing to worry about.

Reply:
This is specifically used instead of RUSTG_OVERRIDE_BUILTINS; the ones that don't work well on Linux aren't included here.